### ISSUE TITLE
Taxon redirect for GBIF identifiers

### ISSUE THREAD

**Issue by egonw** [WDscholia/scholia#2321]
**Describe the solution you'd like**
a redirect for GBIF identifiers ([P846](https://www.wikidata.org/wiki/Property:P846)).

cc @larsgw

**Comment by egonw** [comment]
> I could also use the Hub tool for now.

Yes, that's also what @Daniel-Mietchen has been saying. We still need examples.

And I also wonder if we use this tool for compact identifier-style mappings; I am not sure I checked that already.

**Comment by egonw** [comment]
Yes, that won't do :(

**Comment by larsgw** [comment]
With the query parameters you mean?

**Comment by egonw** [comment]
Yes, that, but also that it is not using the `identifiers.org` or `bioregistry.io` prefixes. It should be something like `https://hub.toolforge.org/gbif:5130682`

**Comment by larsgw** [comment]
https://hub.toolforge.org/gbif:5130682 does work but those prefixes are based on the property label, not registered prefixes.

**Comment by larsgw** [comment]
https://github.com/maxlath/hub/blob/main/scripts/generate_properties_index

**Comment by larsgw** [comment]
I'm content with the Hub for now. Would Scholia be interested in such a 404 page to add missing taxa? The GBIF data is CC-BY 4.0. If not, this issue can be closed.

**Comment by Daniel-Mietchen** [comment]
Yes, it would be great to have a Scholia workflow that assists in setting up taxon items or enriching exiting ones. We should probably go for a CC0 source here, though.
